AGP-Kleros.md

```diff
@@ -0,0 +1,121 @@
+---
+AGP: N/A
+Title: Common court with Kleros
+Author: Clément Lesaege
+(@clesaege)
+Status: Stage III
+Track: Proclamation
+Created: 2019-09-20
+---
+
+# AGP-XX: Common court with Kleros
+
+## Proclamation text
+
+Aragon will work with Kleros for its court system.
+
+A part of the Kleros native token (PNK) would be minted to be put in an ANT bonded curve (this could use some of the work of Aragon Black on batched bonded curves). To avoid gas wars, there will be some initialization period for this bonded curve where everyone can deposit ANT.
+
+In exchange, Aragon would provide a mix of ETH/DAI/ANT. This can be provided to Kleros cooperative or to the Kleros DAO (using Aragon fund management module). The ETH/DAI would guarantee funding of Kleros improvements and of the ecosystem. The ANT would be there to align incentives between both projects.
+
+The exact amount of tokens to be put in the bonded curve or to be exchanged need to be determined by an agreement of both Aragon and Kleros.
+
+The Aragon Association and the Kleros Cooperative will negotiate an agreement. This agreement and the discussion process will be fully transparent and will be proposed to vote on both Aragon and Kleros governance. This can take the form of multiple deals, each providing funding and additional PNK to the bonded curve. Each will be the subject of a governance proposal in both Kleros and Aragon using the Aragon dapp.
+
+
+## Motivation for writing this proclamation
+
+### Background
+
+#### Kleros presentation
+
+Kleros is a decentralized autonomous organization (DAO) providing arbitration services. It is useful for Curated Registries, Oracles and Escrow use cases. It has been live on the mainnet since August 2018 and had its last release in March 2019 allowing specialized courts. It has handled 80 disputes and currently has 3 applications connected to it:
+
+<div style="text-align:center"><img width="115" alt="t2crlogo" src="https://user-images.githubusercontent.com/8873352/63350320-1eed1080-c355-11e9-8b53-e6b20c98814c.png"></div>
+
+
+The [Token² Curated registry](https://tokens.kleros.io), which is a registry of tokens where Kleros jurors solve disputes about token specifics (Project Name, Logo, Ticker and network address). It is made to prevent scam tokens (tokens impersonating projects and giving wrong addresses) and to provide a way for dapps to access a list of tokens and for people to certify information about token characteristics. It has been used by [Ethfinex](http://ethfinex.com/) (rebranding as DeversiFi) in their listing process and has processed more than 2000 ETH of deposits.
```

Comment:
```suggestion
The [Token² Curated registry](https://tokens.kleros.io), which is a registry of tokens where Kleros jurors solve disputes about token specifics (Project Name, Logo, Ticker, and network address). It is made to prevent scam tokens (tokens impersonating projects and giving wrong addresses) and to provide a way for dapps to access a list of tokens and for people to certify information about token characteristics. It has been used by [Ethfinex](http://ethfinex.com/) (rebranding as DeversiFi) in their listing process and has processed more than 2000 ETH of deposits.
```